http/http.go

```diff
@@ -19,34 +22,68 @@ type Server struct {
 
 // Listen starts the http server on the given port.
 func (server *Server) Listen(port int) {
-	r := mux.NewRouter()
-	s := r.PathPrefix(indexPath)
-	s.HandlerFunc(createIndex(server.Index)).Methods(http.MethodPost)
+	router := mux.NewRouter()
+	router.HandleFunc("/{index}/{id}", createIndex(server.Index)).Methods(http.MethodPost)
```

Comment:
This should still be prefixed by something like `/index` or `/api/index`, for namespacing.